src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -1210,14 +1210,14 @@ static void scenery_eyedropper_tool_down(int16_t x, int16_t y, rct_widgetindex w
  * edi : parameter_3
  */
 static void sub_6E1F34(
-    int16_t x, int16_t y, uint16_t selected_scenery, int16_t* grid_x, int16_t* grid_y, uint32_t* parameter_1,
-    uint32_t* parameter_2, uint32_t* parameter_3)
+    int16_t x, int16_t y, uint16_t selected_scenery, CoordsXY& gridPos, uint32_t* parameter_1, uint32_t* parameter_2,
+    uint32_t* parameter_3)
 {
     rct_window* w = window_find_by_class(WC_SCENERY);
 
     if (w == nullptr)
     {
-        *grid_x = LOCATION_NULL;
+        gridPos.x = LOCATION_NULL;
```

Comment:
Use setNull().

Reply:
This is `LOCATION_NULL` which is a `(int16_t)(uint16_t)0x8000` while `setNull` uses `COORDS_NULL` -> `int32_t = 0xFFFF8000`
Should I still use it?

Reply:
Yes, but make sure that anything that tries setting the gridPos to null properly uses setNull().

Reply:
Done.